sdk/search/search-documents/tests.yml

```diff
@@ -13,32 +13,3 @@ jobs:
     parameters:
       PackageName: "@azure/search-documents"
       ResourceServiceDirectory: search
-      Matrix:
-        Linux_Node10:
-          OSVmImage: "ubuntu-18.04"
-          TestType: "node"
-          PublishCodeCoverage: true
-        Windows_Node10:
-          OSVmImage: "windows-2019"
-          TestType: "node"
-        macOS_Node10:
-          OSVmImage: "macOS-10.15"
-          TestType: "node"
-        Browser_Linux_Node10:
-          OSVmImage: "ubuntu-18.04"
-          TestType: "browser"
-          PublishCodeCoverage: true
-        Browser_Windows_Node10:
-          OSVmImage: "windows-2019"
-          TestType: "browser"
-        Browser_macOS_Node10:
-          OSVmImage: "macOS-10.15"
-          TestType: "browser"
-        Linux_Sample10:
-          OSVmImage: "ubuntu-18.04"
-          NodeVersion: "10.x"
-          TestType: "sample"
-      EnvVars:
```

Comment:
my understanding was we needed this section in order to plumb secrets through from `test-resources.json` - is this no longer the case?

Reply:
That was true in an older version of the script and it is no longer the case. This change provided an opportunity to do that cleanup.